pending/configuring-your-dancer-apps.pod

```diff
@@ -0,0 +1,162 @@
+contains this:
+
+        DBIC:
+            default:
+                dsn: dbi:Pg:dbname=my_db
+                schema_class: My::Schema
+                user: dbuser
+                password: dbpass
+                options:
+                    RaiseError: 1
+                    PrintError: 1
+
+Effectively, your running configuration would be this:
+
+    appname: mytestapp
+
+    plugins:
+        Debugger:
+            enabled: 1
+        Auth::Tiny:
+            login_route: '/auth/login'
+        DBIC:
+            default:
+                dsn: dbi:Pg:dbname=my_db
+                schema_class: My::Schema
+                user: dbuser
+                password: dbpass
+                options:
+                    RaiseError: 1
+                    PrintError: 1
+
+If you've converted to Dancer2 from Catalyst, this configuration technique 
+should be familar to you.
+
+=head2 Configuring for Multiple Deployment Environments
+
+In reality, your local development settings will likely differ from shared 
+development server settings, which will likely be different than staging
+or production environment settings. To help you with this, Dancer2 lets you 
+specify your runtime environment when running your application:
+
+    DANCER_ENVIRONMENT=staging plackup bin/mytestapp.psgi
+
+This tells Dancer2 to load F<environments/staging.yml> after F<config.yml> and,
+if it exists, F<config_local.yml>.
+
+As with the previous example, the configuration files are merged in order, with
+settings in F<environments/staging.yml> taking precedence over all others. If
+you had a F<environments/staging_local.yml> file, it would get read and merged
+after all other files.
+
+=head2 One Final Example: Read All the Files!
+
+If you are using YAML-based configuration files only, and are launching an
+application in a production environment, these settings:
+    
+    DANCER_CONFIG_EXT=yml
+    DANCER_ENVIRONMENT=production
+
+will read the following files (for sake of illustration, let's assume you use
+every one of these files):
+
+    appdir/config.yml
+    appdir/config_local.yml
+    appdir/environments/production.yml
+    appdir/environments/production_local.yml
+
+Settings found in the last file merged B<always> takes precedence, followed by 
+settings found in the next most recently read file, and so on.
+
+=head2 Debugging Configuration Issues
+
+If you are ever unsure what order configuration files are being processed in,
+setting the environment variable C<DANCER_CONFIG_VERBOSE=1> prior to 
+launching your application will show you the order in which files have been
+loaded.
+
+=head2 Wrapping Up
+
+Dancer2 developers gained a lot of flexibility in how to to configure their 
+applications in 2016. The core team would like to give a shout-out to Jonathan
+Scott Duff (aka perlpilot) for his work in making this possible.
+
+=head2 Author
+
+Jason A. Crome (CromeDome) C<cromedome@cpan.org>
```

Comment:
Let's add a copyright (@syspete and I waver our copyright with a small note, but feel free to use any other license you wish.)